Cargo.toml

```diff
@@ -5,7 +5,7 @@ members = ["crates/*"]
 # to inherit from this rather than needing to duplicate its content themselves.  Until we get there,
 # keep this in sync with each subpackage `Cargo.toml`'s `package` key.
 [workspace.package]
-version = "0.25.0"
+version = "0.25.2"
```

Comment:
It's fine to leave these as 0.25.0 if you prefer to make sure the Rust components build - this stuff is never exposed outside of Terra nor pushed to any external source, so the numbers don't really matter.

Reply:
reverted in dab0f1f035334ae2ea87c9d23526cb68ff7ccf00